src/ui/units/dash/containers/Dialogs/DialogRelations/types.ts

```diff
@@ -55,6 +57,8 @@ export type ConnectionsData = Array<{
 export type AliasClickHandlerData = {
     showDebugInfo: boolean;
     currentRow: DashkitMetaDataItem;
+    relationText: React.ReactNode | string;
```

Comment:
Doesn't ReactNode include string?

Reply:
According to @types react it doesn't.
<img width="780" alt="image" src="https://github.com/datalens-tech/datalens-ui/assets/98329176/6d62192e-12b5-4a23-8949-8c0b7c0c0964">

Reply:
![2023-10-06_14-06-51](https://github.com/datalens-tech/datalens-ui/assets/56549651/a175c82b-efb0-4ea4-afeb-460fb815d221)
Well hidden

Reply:
fixed